lib/hanami/command_line.rb

```diff
@@ -1,26 +1,28 @@
+require 'hanami/cli'
 require 'thor'
 require 'hanami/cli_base'
 require 'hanami/commands/console'
 require 'hanami/commands/new/app'
 require 'hanami/commands/new/container'
 
 module Hanami
+  module CommandLine
+    include Hanami::Cli
+
+    class Version
+      def call
+        puts "v#{Hanami::VERSION}"
+      end
+    end
+
+    register "version", Version
```

Comment:
@davydovanton for now hanami-cli is very naive, this is a PR for testing the new Cli, I think we shouldn't merge yet.